config/locales/it.yml

```diff
@@ -155,25 +195,138 @@ it:
       spree/zone:
         description: Descrizione
         name: Nome
+        default_tax: Zona di Tassazione di Default
+      spree/adjustment:
+        adjustable: Variabile
+        amount: Quantità
+        label: Descrizione
+        name: Nome
+        state: Stato
+        adjustment_reason_id: Ragione
```

Comment:
This doesn't sound right. "Ragione" is more often meant as "Rationality"...

I'd go with "Motivazione". @alepore what do you think?